src/Lib/Element.php

```diff
@@ -70,7 +82,7 @@ public function getElementName()
      */
     final protected function addElement(Element $element, $isInternal = true)
     {
-        $elementName = $element->getElementName();
+        $elementName = (string) $element->getElementName();
```

Comment:
Considering it's already an Element, can it not ensure elementName is already a stringy?